Header header

uint8 HOME = 0
uint8 MOVE = 1
uint8 PICK = 2
uint8 PLACE = 3
uint8 FINISH = 4

uint8 command
geometry_msgs/Pose to_pose
